integrations/web-chat/introduction.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Webchat and the Chat API"
+sidebarTitle: "Overview"
+description: "Use the PolyAI Chat API to power Webchat."
+---
+
+<Tip>Visit the [API reference](/api-reference/chat) for more detailed information on Chat API response parameters and authentication.</Tip>
+
+Use the Chat API to integrate PolyAI agents into existing <Tooltip tip="Non-voice chat systems, including live in-browser chats and widgets, as well as text and email conversations.">Webchat</Tooltip>
+systems, and use its endpoints to get programmatic access to your automated chat interactions. This API structures non-voice web
+integration conversations, messages, and chat sessions using a simple set of REST endpoints.
+
+The Chat API powers multiple PolyAI Webchat integrations:
+
+- [Email](email)
+- [SMS](sms)
+- [In-app chat](in-app-chat)
+- [WhatsApp](whatsapp)
+
+## Chatting with the API
+
+1. **Initiate conversation**: Send a POST `/chat/create` request with required headers.
+2. **Send and receive messages**: Use the `/chat/respond` endpoint for ongoing interaction.
+3. **Validation**: Requests are validated for API key, token, and payload.
+4. **Conversation termination**: Use the `end_conversation` flag in responses to identify when a session ends.
+
+## Endpoints
+
+### Base URLs
+
+The Chat API is available in the following regions:
+
+| Region      | URL                                  |
+|-------------|--------------------------------------|
+| US          | https://api.us-1.platform.polyai.app |
+| UK          | https://api.uk-1.platform.polyai.app |
+| EUW         | https://api.euw-1.platform.polyai.app |
+
+### API path format
+
+All endpoints follow this structure:
+
+```http
+POST /v1/{account_id}/{project_id}/chat/{operation}
+```
+
+- `account_id`: Your PolyAI account ID, e.g., `ACCOUNT-12345678`.
+- `project_id`: Your PolyAI project ID, e.g., `PROJECT-87654321`.
+
+## User flow
+
+### Pre-usage setup
+
+1. **Request access**: Request Chat API enablement for their project.
+2. **Provision credentials**: A PolyAI engineer generates an API key and connector token.
+3. **Authorization setup**: The API key must include the required permissions:
+
+```json
+[
+  {
+    "Action": "PLATFORM:PROJECT-eb85e04a:POST:/v1/PLATFORM/PROJECT-eb85e04a/chat/create",
+    "Effect": "ALLOW"
+  },
+  {
+    "Action": "PLATFORM:PROJECT-eb85e04a:POST:/v1/PLATFORM/PROJECT-eb85e04a/chat/respond",
+    "Effect": "ALLOW"
+  }
+]
+```
```

Comment:
Are these internal or external docs? - I don't think we need to show our auth structure externally

Reply:
external -- still figuring this stuff out for the first time